src/pages/profile.tsx

```diff
@@ -22,24 +22,28 @@ import useUserContext from 'useUserContext';
 const UserProfile: NextPageWithLayout = () => {
   const [user, setUser] = useUserContext();
   const [name, setName] = useState<string>('');
-  const [show, setShow] = useState(false);
+  const [loaded, setLoaded] = useState(true);
```

Comment:
Use the negated form of `loading` instead. This is the pattern consistently used in other places.

Reply:
done, changed to `notLoaded`